e2e/imgbuild/imgbuild.go

```diff
@@ -121,6 +121,12 @@ func (c *imgBuildTests) nonRootBuild(t *testing.T) {
 			buildSpec: "docker://busybox:latest",
 			sandbox:   false,
 		},
+		// #4517 - CentOS7 sandbox as non-root needs permissions forced
+		{
+			name:      "docker centos7 to sandbox",
+			buildSpec: "docker://centos:7",
```

Comment:
is this specific image needed for this? If there's something special about this image, please document it.

Otherwise, is it possible to use a smaller image for the same test?

Reply:
I am constructing one now, per discussion with Ian K.